pkg/apis/jenkins.io/v1/types.go

```diff
@@ -46,6 +46,8 @@ type SSOSpec struct {
 	URLTemplate string `json:"urlTemplate,omitempty"`
 	// SkipExposeService to avoid using exposecontroller to create ingress rule for proxy
 	SkipExposeService bool `json:"skipExposeService,omitempty"`
+	// SSLInsecureSkipVerify allows the proxy container to connect with a OIDC using selft-sogned certs, this should be used for testing only
+	SSLInsecureSkipVerify bool `json:"sslInsecureSkipVerify,omitempty"`
```

Comment:
Shouldn't this be `tls`?

Reply:
It maps to the same flag that's set on for the proxy https://github.com/rawlingsj/sso-operator/blob/c561ba6b9b3caeaf7f9c19bd9fe90f24003d85dc/pkg/proxy/proxy.go#L325 so maybe `ProxySSLInsecureSkipVerify`?

Reply:
Fair enough :-)